src/platforms/macos/macosutils.mm

```diff
@@ -89,3 +89,42 @@ bool dockClickHandler(id self, SEL cmd, ...) {
 void MacOSUtils::showDockIcon() {
   [NSApp setActivationPolicy:NSApplicationActivationPolicyRegular];
 }
+
+void MacOSUtils::installerStrings() {
+  //% "Mozilla VPN for macOS"
+  qtTrId("macosinstaller.title");
+
+  //% "Unable to install"
+  qtTrId("macosinstaller.unsupported_version.title");
+
+  //% "Mozilla VPN requires Mac OS X 10.6 or later."
+  qtTrId("macosinstaller.unsupported_version.message");
+
+  //% "Previous Installation Detected"
+  qtTrId("macosinstaller.previous_build.title");
+
+  //% "A previous installation of Mozilla VPN exists at /Applications/Mozilla VPN.app. This"
+  //% "installer will remove the previous installation prior to installing. Please back up any data"
+  //% "before proceeding."
+  qtTrId("macosinstaller.previous_build.message");
+
+  //% "You will now be guided through the installation steps for the Mozilla VPN. Thank you for"
+  //% "choosing your VPN from the trusted pioneer of internet privacy."
+  qtTrId("macosinstaller.welcome.message1");
+
+  //% "Click “Continue” to continue the setup."
+  qtTrId("macosinstaller.welcome.message2");
+
+  //% "Success!"
+  qtTrId("macosinstaller.conclusion.title");
+
+  //% "The Mozilla VPN is successfully installed. Go to your Applications folder to open up the VPN
```

Comment:
You're missing quotes here, and this cuts the second part of the string off in the export.